content/en/observability_pipelines/install_the_worker/worker_commands.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Worker Commands
+disable_toc: false
+further_reading:
+- link: "observability_pipelines/install_the_worker/"
+  tag: "Documentation"
+  text: "Install the Worker"
+---
+
+## Run, tap, or top the Worker
+
+Usage example: `observability-pipelines-worker <COMMAND>`
+
+If you are using a containerized environment, use the `docker exec` or `kubectl exec` command to get a shell into the container to run the command.
+
+| Command   | Description                                                                 |
+|-----------|-----------------------------------------------------------------------------|
+| `run`     | Run the observability pipeline worker.                                      |
+| `tap`     | Tap a pipeline to observe events from source or transform components. See [tap options](#tap-options). |
+| `top`     | Lists the components.                                                       |
```

Comment:
More then just listing the components this command provides helpful statistics such as input/output data rates for each component

Reply:
Thanks! Added the info.